cmd/ctr/commands/run/run.go

```diff
@@ -140,9 +140,6 @@ var Command = cli.Command{
 
 		if config {
 			id = context.Args().First()
-			if context.NArg() > 1 {
```

Comment:
I'm not too familiar with urfave cli, but isn't this only checking (positional) arguments, not `--flags`?

Reply:
In this case it checks the number of arguments passed in

Reply:
That feels a bit like a bug if it counts both `--opt foo` (option and value) and `id` as the same (number of arguments)